stemcell_builder/stages/bosh_azure_agent_settings/apply.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+base_dir=$(readlink -nf $(dirname $0)/../..)
+source $base_dir/lib/prelude_apply.bash
+
+# Set SettingsPath but never use it because file_meta_service is avaliable only when the settings file exists.
+cat > $chroot/var/vcap/bosh/agent.json <<JSON
+{
+  "Platform": {
+    "Linux": {
+      "DevicePathResolutionType": "virtio"
+    }
+  },
+  "Infrastructure": {
+    "NetworkingType": "dhcp",
```

Comment:
If you grep bosh-agent source code, there is no NetworkingType. May be you want to set use_dhcp flag on each network: https://github.com/cloudfoundry/bosh-agent/blob/master/settings/settings.go#L124 like AWS CPI does: https://github.com/cloudfoundry-incubator/bosh-aws-cpi-release/blob/ec62d78c16c857f1a6538a9365b4c8f3204d0a57/src/bosh_aws_cpi/lib/cloud/aws/cloud.rb#L784